apps/prairielearn/src/server.js

```diff
@@ -590,6 +590,8 @@ module.exports.initExpress = function () {
     require('./pages/instructorRequestCourse/instructorRequestCourse.js'),
   ]);
 
+  app.use('/pl/navbar', require('./pages/navbar/navbar').default);
```

Comment:
This looks like a new pattern in `server.js`. For other pages we will write out the full route in `server.js` and call the authz middleware from `server.js`, rather than inside the page subrouter. Looking at `pages/navbar` it looks like the two cases (courses and course instances) are handled completely separately (including separate code in the `.html.ts`) so it feels to me like it would be cleaner to have two separate pages here and mount them separately, with the authz middleware called from `server.js`.

Reply:
I changed this around in c6b1ec3, does that look better?

Reply:
Yep, perfect, thanks. I actually hadn't appreciated the fact that the authz middleware wasn't needed for the course route so the extra comments here were very helpful!